app/views/clearance_mailer/change_password.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <p>
   <%= link_to t(".link_text", default: "Change my password"),
-    edit_user_password_url(@user, token: @user.confirmation_token.html_safe) %>
```

Comment:
The token is a `SecureRandom.hex` which can return `0-9` and `a-f`, which are all URL safe characters.

https://github.com/thoughtbot/clearance/blob/b04a0d163d0087b57ec4e27f1eff10b88cc1ec29/lib/clearance/token.rb#L11

https://ruby-doc.org/stdlib-2.7.0/libdoc/securerandom/rdoc/Random/Formatter.html#method-i-hex